src/py/flwr/cli/new/templates/app/code/flwrtune/app.py.tpl

```diff
@@ -0,0 +1,85 @@
+"""$project_name: A Flower / flwrtune app."""
+
+import os
+import warnings
+from datetime import datetime
+
+from flwr_datasets import FederatedDataset
+from hydra import compose, initialize
+from hydra.utils import instantiate
+
+from flwr.client import ClientApp
+from flwr.common import ndarrays_to_parameters
+from flwr.server import ServerApp, ServerConfig
```

Comment:
nit: there must be a blank line b/c from the perspective of someone participating in the challenge, `flwr` is a third-party dependency, but `$import_name` is a first-party dependency:

```suggestion
from flwr.server import ServerApp, ServerConfig

```